docker/compose.devcontainer.yaml

```
@@ -116,31 +127,45 @@ services:
+    volumes:
+      - source: postgres_test_data
+        target: /var/lib/postgresql/data
+        type: volume
 # https://docs.docker.com/reference/compose-file/volumes/
 volumes:
+  api_bash_history:
+  api_node_modules:
+  api_pnpm_store:
   cloudbeaver_data:
+  minio_test_data:
+  postgres_test_data:
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)